### ISSUE TITLE
Question on Handlebars

### ISSUE THREAD

**Issue by bkurowski** [samtecspg/articulate#642]
### Question

- Did you download a release or pull from Master? Which release?
 --> Latest Release 21.2
- What operating system are you running?
--> Ubuntu
- Are you running locally or in the cloud?
-->Locally
- Which browser are you using?
-->Chrome

----

Hi Guys, I have a simple problem, but I have not used handlebars in a while again and I cant get a simple response up. 
Can you please have a look as it might be way faster for you to get over.
I need a simple currency rate checker I have an idea on how to do it, it is just that I am not sure which handlebar will allow me to iterate through the JSON object. I have tested forOwn, iterate but always getting Sorry Can you rephrase that?, I might be setting the handlebars in wrong.   

My plan was to use this api : _**https://api.exchangeratesapi.io/latest**_

And i was planning to do something like :

`{{forOwn webhookResponse}} {{#contains rates ../slots.currencyIn.original}} {{this}} {{/contains}} `

So if the user specifies the currency code this get passed and iterates through JSON looking for that one and spits out its value. That was my intention but seems like I did not go too far 👎 

I am getting this error in console after running it. 
![image](https://user-images.githubusercontent.com/40358189/54218300-37731800-44e5-11e9-9156-1534e960c3a7.png)

Can anyone throw in some idea ? 

Many Thanks !

**Comment by wrathagom** [comment]
I think you should be able to do `{{lookup webhookResponse.rates slots.currencyIn.original}}`

**Comment by bkurowski** [comment]
That is it, thank you !
